### DISCUSSION TITLE
Is Dry run only working in release?

### DISCUSSION THREAD

**Discussion by alanblu** [semantic-release/semantic-release#3220] [Q&A]
Can I run semantic release with dry run in another repository beside release?

**Comment by travi** [comment]
It's not clear what your goal is. semantic-release analyzes the commits from the repository it is configured to make releases for. It had to be able to check out the commits to analyze them, so it would be a strange use case to do that from a different repository

**Reply by alanblu** [reply]
I need the Tag (version) to compile in a previous pipeline before semantic release

**Reply by travi** [reply]
why is that something you believe you need to do? it is recommended to coordinate your entire release process through semantic-release plugins. is there a reason that is not an option for you?

**Reply by alanblu** [reply]
Thanks for the answer!

**Reply by alanblu** [reply]
I wanted to get the Version to test some cases related to Version in the software (is a microcontroller so I need to upload it and compile it before testing. But I found other solution, thank you very much for your answer!

**Comment by alanblu** [comment]
I solved it in another way
